archive_viewer/widgets/item_delegates.py

```diff
@@ -99,7 +99,6 @@ def initStyleOption(self, option: QStyleOptionViewItem, index: QModelIndex):
         """Initialize a QComboBox for use in the Table View."""
         if index.row() >= len(self.editor_list):
             editor = QComboBox()
-
```

Comment:
We can leave some whitespace. This will naturally break up the sections of the function and makes it easier to read.